hnaimm/shift-smart
# FEAT: Authentication and User Verification

### 🛠 Changes Summary

- Enable login and logout.
- Enable registration with email address verification.
- Add "Verification" table to DB and add is_verified column to "User" table.
- Make wishlist page protected route (only logged in users can access the page)
- 
### 👍 How to test

- Go to login page through navbar or by going to wishlist where you will be redirected automatically to login page.
- Press on "Register" tab.
- Fill in your information.
- Check your inbox for receive verification email.
- Press on the button in email. you will be redirected to the website where you're logged in and can access wish list now.



### 🔳 Changes Scope


### ✨ Changes Type

- [ ] Existing Feature

### ⌛ What's Pending

_Mention whether anything task related to this feature/fix is still pending_

### 📸 Screenshots

![image](https://github.com/user-attachments/assets/d43d2d3c-090f-43d8-973c-b8e02b318bba)

![image](https://github.com/user-attachments/assets/8736ee53-41d4-46c6-992c-4e7bdc699269)

### ✍ Additional Info

_Any additional details_
